client-go/src/main/iotdb_session/session.go

```diff
@@ -0,0 +1,290 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package iotdbSession
+
+import (
+	"context"
+	"fmt"
+	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/yanhongwangg/go-thrift/rpc"
+	"log"
+	"net"
+	"os"
+	"strings"
+	"time"
+)
+
+const protocolVersion = rpc.TSProtocolVersion_IOTDB_SERVICE_PROTOCOL_V3
+
+var client *rpc.TSIServiceClient
+var sessionId int64
+var isClose bool = true
+var trans thrift.TTransport
+var err error
+var requestStatementId int64
+
+func (s *Session) Open(enableRPCCompression bool, connectionTimeoutInMs int) error {
+	dir, _ := os.Getwd()
+	os.Mkdir(dir+"\\logs", os.ModePerm)
```

Comment:
I think the logs should be redesign,
I suggest using the log : https://github.com/sirupsen/logrus

Reply:
good idea!!!